examples/tts/conf/rad-tts_dec.yaml

```diff
@@ -8,7 +8,7 @@ export_dir: ???
 sup_data_path: ???
 sup_data_types: ["log_mel", "align_prior_matrix", "pitch", "voiced_mask", "p_voiced", "energy"]
 
-whitelist_path: "nemo_text_processing/text_normalization/en/data/whitelist/tts.tsv"
+whitelist_path: null # https://github.com/NVIDIA/NeMo-text-processing/blob/main/nemo_text_processing/text_normalization/en/data/whitelist/tts.tsv default will be used
```

Comment:
seems to conflict with the above changes. Are you planning to host whitelist files in NeMo repo or the new NeMo-Text-Processing repo? why do we guide users to assign the full url instead of local files?

Reply:
The whitelist file that was used in most  TTS yaml files is the default file used for TN. Using `null` here will do exactly the same thing as was done before.`whitelist_path` is still there in the config if users want to pass a custom file. And I added a URL here for those who want to see the whitelist format.
`libri_speech.tsv` is added only because it is used during preprocessing. I can double check by libri_speech should be a subset of the default whitelist, so maybe just use "null" for lj_preprocessing?